english/light-fire/sprinkler/wonka-construction/wonka-construction.md

```diff
@@ -7,4 +7,4 @@ Start eating the graham grackers?
 
 Ask the Wonk constuction work crew to identify themselves?
 
-Fill the chocolate pool with graham crackers?
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.